fdm-app/app/routes/farm.$b_id_farm.$calendar.measures._index.tsx

```
@@ -0,0 +1,745 @@
+                fd.append("m_start", data.m_start)
+                if (data.m_end) fd.append("m_end", data.m_end)
+                else fd.append("m_end", "")
+                fetcher.submit(fd, { method: "post", action })
+                onClose()
+            },
+        },
+    })
+
+    // Reset form when the row or mode changes
+    const { reset } = form
+    useEffect(() => {
```

Biome (2.4.15)

[error] This hook specifies a dependency more specific than its captures: row?.m_id

(lint/correctness/useExhaustiveDependencies)